data/hl7-sample/test.txt

```diff
@@ -0,0 +1 @@
+this is a test text file
```

Comment:
Can we delete this file? @amathew8 make sure you don't add/commit files that get added to the `/data` folder if you don't want them to show up on github.

Reply:
I changed the `.gitignore` to only ignore `/data/hl7-uploads` so that we could keep test files in `/data/hl7-sample` and this file is being used in the tests to test the upload endpoint.